From dd454f81f3a9da42502ef4ac4120222416db6c58 Mon Sep 17 00:00:00 2001
From: Junki Kato <junki.kato.xk@renesas.com>
Date: Tue, 19 Dec 2017 13:11:43 +0900
Subject: [PATCH 249/909] phy: rcar-gen3-usb2: Add a compatible string for
 r8a77995

commit 6380ce0f1856bb47b552610c9b07a5a0d9876396 from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git

This patch adds a compatible string for r8a77995.

Signed-off-by: Junki Kato <junki.kato.xk@renesas.com>
Signed-off-by: An Huynh <an.huynh.uj@renesas.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/phy/renesas/phy-rcar-gen3-usb2.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/drivers/phy/renesas/phy-rcar-gen3-usb2.c b/drivers/phy/renesas/phy-rcar-gen3-usb2.c
index fb8f05e39cf7..c9e7fe651cb2 100644
--- a/drivers/phy/renesas/phy-rcar-gen3-usb2.c
+++ b/drivers/phy/renesas/phy-rcar-gen3-usb2.c
@@ -400,6 +400,9 @@ static const struct of_device_id rcar_gen3_phy_usb2_match_table[] = {
 		.compatible = "renesas,usb2-phy-r8a77965",
 		.data = (void *)RCAR_GEN3_PHY_HAS_DEDICATED_PINS,
 	},
+	{
+		.compatible = "renesas,usb2-phy-r8a77995",
+	},
 	{
 		.compatible = "renesas,rcar-gen3-usb2-phy",
 	},
-- 
2.17.1

